docs/1_Getting_Started.md

```diff
@@ -114,6 +114,35 @@ caliper benchmark run --caliper-workspace ./packages/caliper-samples --caliper-b
 
 The files present in the `caliper-samples` directory may be modified or added to, in order to perform the desired benchmark. Before adding a benchmark, please inspect the example benchmark content and structure; you will need to add your own configuration files for the blockchain system under test, the benchmark configuration, smart contracts, and test files (callbacks) that interact with the deployed smart contract.
 
+#### Caliper Flow Control
+
+The default Caliper lifecycle is:
+- Start SUT (using a `start` command in the network configuration file)
+- Initialize the SUT
+- Install smart contracts
+- Perform the benchmark test
+- Tear down SUT (using an `end` command in the network configuration file)
+
+In some cases it is desirable to run a subset of the complete Caliper lifecycle. It is possible to do this by using command line flags:
```

Comment:
Proposed (raw) text to be more general:

"It is possible to do this by setting one or more of the following keys through the [runtime configuration mechanism.]({{ site.baseurl }}{% link docs/Runtime_Configuration.md %})"